test/OpenFeature.Contrib.Hooks.Otel.Test/OpenFeature.Contrib.Hooks.Otel.Test.csproj

```diff
@@ -4,9 +4,9 @@
         <ProjectReference Include="..\..\src\OpenFeature.Contrib.Hooks.Otel\OpenFeature.Contrib.Hooks.Otel.csproj" />
     </ItemGroup>
 
-    <ItemGroup>
-      <PackageReference Include="OpenTelemetry" Version="1.4.0" />
-      <PackageReference Include="OpenTelemetry.Exporter.InMemory" Version="1.4.0" />
+    <ItemGroup>
+      <PackageReference Include="OpenTelemetry" Version="1.4.0" />
+      <PackageReference Include="OpenTelemetry.Exporter.InMemory" Version="1.4.0" />

```

Comment:
Please revert this change because it will bump the version of the OTel hook.

Reply:
looks like VS changed smth in formatin. I'll revert

Reply:
@beeme1mr @vpetrusevici I dont think this will impact any releases. This is under the `test` folder. Release please only watches `src`: https://github.com/open-feature/dotnet-sdk-contrib/blob/d8cac7fa6f757f6d62c9648eb249d80528f2b337/release-please-config.json#L5

Reply:
@toddbaert updated. I hope it's right

Reply:
I still see whitespace changes but I'm not worried about them. They should not impact anything, since they are not in the `src/` dir.